### DISCUSSION TITLE
Create operator to remove undefined props

### DISCUSSION THREAD

**Discussion by renatodysouza** [ReactiveX/rxjs#6915] [Ideas / Feature request]
It is easy to resolve, but if this operator could exist.

Example
const myFormValue = {
   name: 'John',
  age: '',
  city: ''
};

of(myFormValue ).pipe(removeUndefinedProps).subscribe(val => console.log(val));  //  { name: john }

**Comment by josepot** [comment]
It's probably easier to define a mapper function like `removeFalsyProps` and then use it with `map`:

```js
const removeFalsyProps = (obj) => Object.fromEntries(
  Object.entries(obj).filter(([, val]) => val)
);

of(myFormValue).pipe(
  map(removeFalsyProps)
)
```

**Reply by renatodysouza** [reply]
Hi Josepot, Everthing ok, I hope so. I agree with with you, but the idea is put this feature built-in .  Normal all feature are easy, the question is   resolve not the imperative way, but declarative way.

Best regards,
